### PR TITLE
feat: introduce order book top channel

### PR DESCRIPTION
**Pull request by udonyx**

### CODE REVIEW

File: protos/onyx_otc/v2/responses.proto [hunk 1]
```diff
@@ -69,6 +69,14 @@ message OtcQuote {
   string product_symbol = 7;
 }
 
+message OrderBookTop {
+  Decimal buy = 1;
```
Comment (lsbardel) [inline protos/onyx_otc/v2/responses.proto:73]:
```suggestion
  PriceAmount buy = 1;
```

File: protos/onyx_otc/v2/responses.proto [hunk 2]
```diff
@@ -69,6 +69,14 @@ message OtcQuote {
...
+  Decimal sell = 2;
```
Comment (lsbardel) [inline protos/onyx_otc/v2/responses.proto:74]:
```suggestion
  PriceAmount sell = 2;
```

File: protos/onyx_otc/v2/responses.proto [hunk 3]
```diff
@@ -69,6 +69,14 @@ message OtcQuote {
   string product_symbol = 7;
 }
```
Comment (lsbardel) [inline protos/onyx_otc/v2/responses.proto:70]: message PriceAmount {
    Decimal price = 1;
    Decimal amount = 2;
}

**Review by lsbardel — CHANGES_REQUESTED** [review]

File: protos/onyx_otc/v2/responses.proto (hunk 3 above)
Comment (udonyx) [reply protos/onyx_otc/v2/responses.proto:76]: done

File: protos/onyx_otc/v2/responses.proto (hunk 1 above)
Comment (udonyx) [reply protos/onyx_otc/v2/responses.proto]: Done

**Pull request merged**